pyverdict/pyverdict/jvm.py

```diff
@@ -0,0 +1,17 @@
+import time
+import sys
+import subprocess
+from py4j.java_gateway import JavaGateway
+import pyverdict
+
+class JVM:
+    def __init__(self):
+        subprocess.Popen(['java', '-cp', pyverdict.JAR_PATH, 'org.verdictdb.VerdictGateway'])
+        time.sleep(1)
+        self.gateway = JavaGateway()
+
+    def connect(self):
+        return self.gateway
+
+    def stop(self):
```

Comment:
Change the method name to `close()`. Register it to `atexit`: https://docs.python.org/2/library/atexit.html